scripts/zones/Temple_of_Uggalepih/npcs/_mf6.lua

```diff
@@ -0,0 +1,36 @@
+-----------------------------------
+-- Area: Temple of Uggalepih
+--  NPC: _mf8 (Granite Door)
+-- Note: Opens with Uggalepih Key
+-- !pos -208 -1.89 -20
+-----------------------------------
+local ID = require("scripts/zones/Temple_of_Uggalepih/IDs")
+require("scripts/globals/npc_util")
+-----------------------------------
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+    if npcUtil.tradeHasExactly(trade, xi.items.UGGALEPIH_KEY) then
+        player:confirmTrade()
+        player:messageSpecial(ID.text.YOUR_KEY_BREAKS, 0, xi.items.UGGALEPIH_KEY)
+        npc:openDoor(6.5)
+    end
+end
+
+entity.onTrigger = function(player, npc)
+    if player:getXPos() >= -208 then
+        player:messageSpecial(ID.text.THE_DOOR_IS_LOCKED, xi.items.UGGALEPIH_KEY)
+    else
+        npc:openDoor(11)
+    end
+
+    return 1
```

Comment:
Is this a carryover from old code?  From what I can see, the only retval from onTrigger that causes action in core is -1

Reply:
Removed that, definitely a copy paste on my end from Wings that I should have removed.